### PR TITLE
IDE-1748 Add maven sub projets to working set

### PR DESCRIPTION
**Pull request by ericmin**

### CODE REVIEW

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/PluginsSDKProjectProvider.java [hunk 1]
```diff
@@ -208,6 +212,8 @@ public IStatus doCreateNewProject( NewLiferayPluginProjectOp op, IProgressMonito
         // need to update project name incase the suffix was not correct
         op.setFinalProjectName( newProject.getName() );
 
+        projectNames.insert().setName( op.getFinalProjectName().content() );
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/PluginsSDKProjectProvider.java]: Good job this is the missing piece, now both maven and SDK providers use the new projectNames and also the value added to finalProjectName will get correctly added here.

File: tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/wizard/NewLiferayPluginProjectWizard.java [hunk 2]
```diff
@@ -154,23 +160,39 @@ protected void performPostFinish()
...
+                addToWorkingSets( project );
+            }
+            catch( Exception ex )
+            {
+                ProjectUI.logError( "Unable to add project to working set", ex );
+            }
         }
 
-        openLiferayPerspective( project );
+        openLiferayPerspective( projects.get(0) );
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/wizard/NewLiferayPluginProjectWizard.java]: Lets add the projects.get(0) to a local final variable so we don't call it 5 times.

**Comment by gamerson** [comment]
@ericmin  can you also add a unit test for this one?  note: you can't add a test that things get added to working set since that involves the UI.  but you can change the value of the projectNames list after running the test and assert that it contains the values you expect for all of these things:

SDK portlets (single project that has value or getFinalProjectName is used in projectNames list)
Maven Portlets (single project in in projectNames)
Maven Service Builder Portlets (multiple projects are in projectNames)

Closing see comments, thanks.

**Comment by ericmin** [comment]
Sure. Will do. Thanks Greg.

**Pull request closed without merging**